.github/workflows/ci.yml

```diff
@@ -174,9 +174,7 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - config-empty
```

Comment:
This probably wasn't CDH5

Reply:
You're right, this should have gone in the other commit to remove pre Hive 3.x